modules/svdb/query/main.nf

```diff
@@ -63,12 +63,11 @@ process SVDB_QUERY {
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}_query.vcf
+    touch ${prefix}_ann_svdbq.vcf
```

Comment:
Why this change the output channel declares `*_query.vcf` right?

Reply:
Good catch! Fixed.